index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./versions/1.0.0/schema.json')
```

Comment:
I'm thinking eventually this can be replaced with a symlinked `latest` or `current` that is updated during a release/version bump.